types/dropzone/index.d.ts

```diff
@@ -131,7 +131,7 @@ declare namespace Dropzone {
         sending?(file: DropzoneFile, xhr: XMLHttpRequest, formData: FormData): void;
         sendingmultiple?(files: DropzoneFile[], xhr: XMLHttpRequest, formData: FormData): void;
 
-        success?(file: DropzoneFile, response: Object | string): void;
```

Comment:
![image](https://user-images.githubusercontent.com/1018543/69856263-33cc2f00-12c8-11ea-886d-da4cf768e2c1.png)
The file has been uploaded successfully. Gets the server response as second argument. (This event was called finished previously)
So the option `success` should still have `response` as 2nd argument
It's just that the default option does not need `response` at all

Should revert change for test too

Reply:
Hey! Thanks for your review!

I believe this is not the case. What you're commenting on is the _configuration option_ in the types, but the screenshot refers to the _events_, see [here](https://gitlab.com/meno/dropzone/blob/abe7a2af6873feaacec407c0c64cc3842211ad8f/src/dropzone.js#L870-876) for the config option which indeed does only receive `file`. The event list receives both, `file` and `response`, see [here](https://gitlab.com/meno/dropzone/blob/abe7a2af6873feaacec407c0c64cc3842211ad8f/src/dropzone.js#L2239).

Sadly the dropzone.js website is missing the config option in their documentation. But maybe this is indeed on purpose as the [event list documentation](https://www.dropzonejs.com/#event-list) states:

> Do **not** overwrite those as configuration options, unless you know what you're doing.